merge_static.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
```

Comment:
Is the root directory the best place for merge_static.sh? Perhaps in `src` instead?

Reply:
I don't care too much either way